src/RadioButtons/index.js

```diff
@@ -100,7 +100,7 @@ const RadioButtons = ({
                 role="presentation"
                 className={cc([
                   "nds-radio",
-                  { "narmi-icon-check": kind === "card" },
+                  { "narmi-icon-check": ["card", "checkmark"].includes(kind) },
```

Comment:
Preface: this is an absolute nit, pr looks awesome!

`includes` checks for string elements in an array which containing a certain string value. This works well when we're checking for `kind` in a unique array of strings, but what if in the future, we want to add another`narmi-icon-check` that includes the word "card" or "checkmark"?